magellan-library/src/main/java/com/wealthfront/magellan/view/DialogComponent.kt

```diff
@@ -37,9 +43,13 @@ class DialogComponent @Inject constructor() : LifecycleAware {
     this.context = null
   }
 
+  fun getDialog(): Dialog? {
+    return dialog
+  }
```

Comment:
nit: make the `var dialog` public with a `private set`

Reply:
d